src/features/staking/stCELO/StakeForm.tsx

```diff
@@ -209,15 +210,8 @@ function validateForm(
     }
   }
 
-  // Ensure user isn't trying to unlock CELO used for staking
+  // Ensure user isn't trying to unlock stCELO used for staking
   if (action === LiquidStakeActionType.Unstake) {
-    if (stCELOBalances.lockedVote > amountWei) {
-      toast.warn(
-        'Locked funds that have voted for a governance cannot be unlocked until the proposal is resolved.',
-      );
-      return { amount: 'Locked funds in use' };
-    }
```

Comment:
my testing this was not needed as the real check is below and incorrect (you couldnt withdraw if less than how much was voting was being unstaked)